fuzzers/utils.py

```diff
@@ -142,3 +149,28 @@ def get_dictionary_path(target_binary):
                                      options_file_path)
                 return dictionary_path
     return None
+
+
+def set_default_optimization_flag_if_needed(env=None):
```

Comment:
I don't really know why the "if_needed" part is useful.
I think this is good. It changes the behavior on OSS-Fuzz (where O1 is already in CFLAGS) and makes it inconsistent with FTS benchmarks.

Reply:
We were resetting flags anyway in set_no_sanitizer_compilation_flags, so optimization flag was always set to -O3. but fixed this function to just add optimization flags always, you are right, that enforces the optimization always (which we want).